public/js/charts/trendsCoverage.js

```diff
@@ -0,0 +1,173 @@
+// NOTE: requires('sheets');
+// NOTE: requires('aq');
+
+window.aq.charts.trendsCoverage = {};
+
+window.Chart.defaults.global.defaultFontSize = 10;
+
+const acquisition = document.getElementById('acquisition');
+
+async function handleLocationChange() {
+  const nodes = await window.aq.getNodes();
+  const countyCitiesMap = await window.sheets.getCountyCitiesMap();
+  function getAvg(sensor) {
+    const total = sensor.reduce((acc, c) => acc + c, 0);
+    return total / sensor.length;
+  }
+  // Format dates from nodes
+  function formatDate(date, langCode) {
+    const dateObj = new Date(date);
+    const day = dateObj.toLocaleString(langCode, { day: '2-digit' }); // DD
+    const month = dateObj.toLocaleString(langCode, { month: 'short' }); // MMM
+    return `${day} ${month}`;
+  }
+  // get chartlabels for the week in an array
+  const labels = [...Array(7)]
+    .map((_, i) => {
```

Comment:
```suggestion
  Array.from({ length: 7 }, (_, i) => {
```

Can be combined [without creating an intermediate array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#description)